lib/ex_unit/lib/ex_unit/case.ex

```diff
@@ -316,7 +319,19 @@ defmodule ExUnit.Case do
       Module.put_attribute(mod, :ex_unit_test_names, Map.put(test_names, name, true))
     end
 
-    Module.delete_attribute(mod, :tag)
+    Enum.each [:tag | registered_attributes], fn(attribute) ->
+      Module.delete_attribute(mod, attribute)
+    end
+  end
+
+  @doc """
+  Registers a new attribute to be used during `ExUnit.Case` tests.
```

Comment:
The first paragraph of docs should always be a quick summary. More info here: http://elixir-lang.org/docs/master/elixir/writing-documentation.html